### PR TITLE
Make list of GitHub repos searchable in project import

### PR DESCRIPTION
**Pull request by sandipbhuyan**
**# Resolving issue** #196 

Now selection of git project will have a UI with search functionality. A user directly can search over a number of projects. 
I have added and imported two libraries of bootstrap-select i.e bootstrap-select.min.css and bootstrap-select.min.js in base.html.twig.
I have modified the drop down section's z-index to over ride the z-index property of navbar.
The UI will look like:
screen shot 2018-03-02 at 1 51 54 pm

### CODE REVIEW

**Comment by sandipbhuyan** [comment]
@agathver and @imphil I have raised a PR against the issue #196.

File: site/app/Resources/views/base.html.twig [hunk 1]
```diff
@@ -134,7 +135,7 @@
   </footer>
 
   {% block javascripts %}
-    {% javascripts filter="?jsqueeze" output="js/app_body.js" '@js_body_end' %}
+    {% javascripts filter="?jsqueeze" output="js/app_body.js" '@js_body_end' 'assets/js/bootstrap-select.min.js' %}
```
Comment (agathver) [inline site/app/Resources/views/base.html.twig]: Should this be present in every page load?

I would rather prefer it to be included in `ProjectRepoBundle::Project::new.html.twig`

File: site/src/Librecores/ProjectRepoBundle/Resources/views/Project/new.html.twig [hunk 2]
```diff
@@ -73,7 +73,10 @@ other information about your project automatically.</p>
             <p>You don't have access to any repository on GitHub (or haven't granted us access).
             Please add the project to LibreCores by entering the Git repository URL manually.</p>
           {% else %}
-            {{ form_row(form.githubSourceRepo, {'label': 'Select a project from your GitHub account'}) }}
+            <div class="form-group">
+              {{ form_label(form.githubSourceRepo,'Select a project from your GitHub account') }}
+              {{ form_widget(form.githubSourceRepo,{'attr': {'class': 'selectpicker','data-live-search':'true'}}) }}
```
Comment (imphil) [inline site/src/Librecores/ProjectRepoBundle/Resources/views/Project/new.html.twig]: two nits: please add a space after the comma, e.g. `form.githubSourceRepo, {'attr':`

File: site/app/Resources/views/base.html.twig (hunk 1 above)
Comment (imphil) [reply site/app/Resources/views/base.html.twig]: yes, please make it only for the "new project" page.

**Comment by sandipbhuyan** [comment]
@agathver @imphil I have made the changes in which i imported both the css and js file in new.html.twig and i have added the un-minified file of js and css

**Comment by imphil** [comment]
Great, thanks! Rebased/merged via a63aa8abb7371ee6b6b583e870d6c46a7a6908ad

**Pull request closed without merging**